### PR TITLE
added in tables to help generate metadata

### PR DESCRIPTION
**Pull request by hswerdfe**
first draft of the tables that will generate the metadata.

### CODE REVIEW

**Comment by DougManuel** [comment]
Suggestions:

- add `variableLabel` a human=readable short label. e.g. variableName = 'sampleID' variableName = 'Sample ID".
- add `tableLabel` — same concept as variableName.

File: Variable.csv [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+Sample,dateTimeEnd,,datetime,"For integrated time average samples this is the�date, time and timezone�the sample was finished being taken."
+Sample,type,,category,Type of sample.
+Sample,typeOther,,string,Description for other type of sample not listed in
+Sample,collection,,category,Method used to collect the data.
+Sample,collectionOther,,string,Description for other type of method not listed in�collection.
+Sample,preTreatment,,boolean,Was the sample chemically treated in anyway with the addition of stabilizers or other?
+Sample,preTreatmentDescription,,string,If�preTreatment�then describe the treatment that was performed.
+Sample,pooled,,boolean,"Is this a pooled sample, and therefore composed of multiple child samples obtained at different sites?"
+Sample,children,,string,If this is a sample with many smaller samples either because of pooling or sub-sampling this indicates�a comma separated list of child sampleID's.
+Sample,parent�,,string,If this sample has been pooled into one big sample for analysis this indicates the�sampleID of the larger pooled sample.
```
Comment (DougManuel) [inline Variable.csv:15]: Looks like there are a bunch of special characters like, 'parent�' that are suppose to be there??

**Review by DougManuel — CHANGES_REQUESTED** [review]
Looks good.

For the record, I suggest that we have category values.
We may want `notes`. 
We may want to add two additional tables, but we can discuss for a later PR:
- a table to describe the metadata. 
- a table to describe a wastewater database. See a few attributes that are covered in other metadata formats like DDI and Dublin Core. https://ddialliance.org/resources/ddi-profiles/dc
We may

File: Variable.csv (hunk 1 above)
Comment (hswerdfe) [reply Variable.csv]: thanks @DougManuel looking into it

**Comment by hswerdfe** [comment]
@DougManuel I am confused by what you want re: `notes`  there is already a `desc`. 

I also don't know what you mean for the other two comments, still looking into the special characters issue.

**Comment by hswerdfe** [comment]
@DougManuel I fixed the special charter I think.

**Comment by DougManuel** [comment]
I added the header and examples of labels. 
notes in addition to desc. Notes would be additional context. It wouldn't be necessary for all variables, however, some descriptions are really descriptions, they are more like notes. e.g.

`-   **accessToPublic**: If this is 'no', this data will not be available to the public. If missing, data will be available to the public.`

becomes:
`variableName` = accessToPublic
`variableLabel` = "Data access to public"
`variableDesc` = "Flag indicating whether the data custodian allows public access to the data field."
`variableNotes` = "If this is 'no', this data will not be available to the public. If missing, data will be available to the public."

**Comment by DougManuel** [comment]
I added the header and examples of labels. I filled in just a few examples.

**Comment by DougManuel** [comment]
All the Boolean variables should probably be in `VariableCategory.csv` to indicate whether the response is 'yes/no', 'true/false', 'Yes/No' or whatever we want.

**Comment by DougManuel** [comment]
Hmm. If we want to be picky, all the tables should be pural. 
- `Table.csv` —-> `Tables.csv`

**Comment by DougManuel** [comment]
VariableCategory.csv:
-  should also have labels. In some situations the late may be identical to the description. I've commonly used the term "catLabelLong" and "catLabelShort" instead of the term desc.
- change name from `variableValue` to `variableCat`??

**Comment by hswerdfe** [comment]
Now 
 * using plural tables
 * has french and english labels
 * generated SQL statements have refs,

**Comment by DougManuel** [comment]
Excellent implementation of autogeneration of metadata.md. 

1) We will need to find a home for a few paragraphs at the bottom of the old metadata.md. Maybe a modified and renamed `templates_and_input.md`.
2) French version of the autogenerated metadata.md. Great inclusion BTW.

Suggest we merge after we sort out these two issues.

3) There are a bunch of little missing or extra periods and minor edits. We can clean these up later. 
4) We can discuss the remaining notes at a later time, including short variable and category labels, notes, tables to describe a WW database.

**Review by DougManuel — CHANGES_REQUESTED** [review]
1) We will need to find a home for a few paragraphs at the bottom of the old metadata.md. Maybe a modified and renamed templates_and_input.md.
2) French version of the autogenerated metadata.md? Great inclusion BTW.

**Comment by hswerdfe** [comment]
I am not sure if we need a french version of the auto generated md file.
I was thinking the labels were just for the template.

**Review by DougManuel — CHANGES_REQUESTED** [review]
Looks good, but I can't find these two sections in the new version of the metadata.md file:

https://github.com/Big-Life-Lab/covid-19-wastewater/blob/main/metadata.md#naming-conventions

https://github.com/Big-Life-Lab/covid-19-wastewater/blob/main/metadata.md#naming-conventions

**Comment by hswerdfe** [comment]
added naming conventions section.

**Review by DougManuel — APPROVED** [review]

**Comment by hswerdfe** [comment]
Approved! WOOHOO!

**Pull request merged**